### DISCUSSION TITLE
Static and Dynamic Rendering

### DISCUSSION THREAD

**Discussion by cvpfus** [vercel/next.js#70501] [Help]
### Summary

I am currently studying this tutorial
https://nextjs.org/learn/dashboard-app/static-and-dynamic-rendering

In that chapter, it mentions that there are two limitations of the current setup:
1. The data requests are creating an unintentional waterfall.
2. The dashboard is static, so any data updates will not be reflected on your application.

I am asking about number 2, which mentions that the dashboard is static. However in the following explanation
https://nextjs.org/learn/dashboard-app/static-and-dynamic-rendering#simulating-a-slow-data-fetch

It mentions that "The dashboard application we're building is dynamic."

So, which one is correct? I am having a hard time understanding the difference between static and dynamic rendering. (I am not a native English speaker, so maybe that is the problem). Can anyone help explain it?

**Comment by icyJoseph** [comment]
That chapter is showing why, what has been currently made, has two issues, and how, the text of the chapter, and future chapters, will fix them. 

So these are the `from static, to dynamic` first steps.

> It mentions that "The dashboard application we're building is dynamic."

Yes, and so far, you have done everything static, which is just step one, now comes step two, to make it dynamic.
